apps/docs-snippets2/src/wallets/fuel-instantiation-options.ts

```diff
@@ -0,0 +1,19 @@
+// #region fuel-options-connectors
+import { FuelConnector, Fuel } from 'fuels';
+
+class WalletConnector extends FuelConnector {
+  name: string = 'My Wallet Connector';
+}
+
+const defaultConnectors = (_opts: {
+  devMode: boolean;
+}): Array<FuelConnector> => [new WalletConnector()];
```

Comment:
This function was to mock out the `fuel/connectors`; `defaultConnectors` method (see [here](https://docs.fuel.network/docs/fuels-ts/wallets/connectors/#connectors-2)).

Not entirely sure how we'd do this with the new way of snippets

Reply:
Upon closer consideration we should actually use `@fuels/connectors` dep here, if it causes an issue then we would at least be made aware of it, but we want our docs to be accurate.

c28b6b22b

Reply:
Sounds reasonable to me 🚀

Reply:
Looks like the reason it was mocked in the first place was due to the dynamic importation of the crypto peer dep which causing [test failures](https://github.com/FuelLabs/fuels-ts/actions/runs/11778333665/job/32804587979?pr=3365#step:6:2012) now 

Probably best we stick with the mocked dep